### PR TITLE
feat(academy): add part1.1

### PR DESCRIPTION
**Pull request by MathieuConte**
Minor change : 

I've just removed the image "SUM UP"
We might design it before adding to the page.
sum up

You can review it 👍

## Summary by CodeRabbit

- **Documentation**
	- Introduced a new "Protocol concepts" section to enhance user understanding of fundamental principles.
	- Updated the "Resources" documentation to clarify the concepts of Digital Resources and Services, and their roles in the protocol.

### CODE REVIEW

**Comment by ccamel** [comment]
@MathieuConte I have slightly adjusted the hierarchy of the titles to align with the expectations set by Docusaurus. We should now be ok.

image

**Review by ccamel — APPROVED** [review]
:+1: thanks

**Comment by MdechampG** [comment]
Thanks @MathieuConte ! 
I changed the content from the page "Protocol-concepts"which is the part 1.4 to the page "Resources" which is the 1.1 
Thus, I readjusted the hierarchy of the titles. 

Could you change the name of the illustrations (protocol-concept -> resources) please ? 

As for the illustration, could you create a dedicated issue ?

**Review by amimart — APPROVED** [review]

**Comment by MathieuConte** [comment]
Ok I've modified the name of illustrations and I'll create a issue for the new illustration

**Comment by MdechampG** [comment]
Thanks @MathieuConte you can merge 🚀

**Review by tpelliet — APPROVED** [review]

File: docs/academy/resources.md [hunk 1]
```diff
@@ -3,3 +3,105 @@ sidebar_class_name: hidden
...
+
+### Standardization Efforts
+
+To foster interoperability and consistency, the OKP4 protocol encourages adherence to established ontological standards. By aligning with widely accepted frameworks, such as RDF (Resource Description Framework) or OWL (Web Ontology Language), the protocol ensures a harmonized representation of knowledge across diverse resources.
+
+### Practical Application
+
+In practical terms, the semantic description facilitated by ontology within the OKP4 protocol extends to various domains. For instance, in the AI sector, ontology can illuminate the intricate relationships between different types of AI algorithms, datasets, and model architectures. This semantic richness provides a comprehensive understanding of the AI landscape, enabling users to navigate and leverage diverse AI resources with greater insight and efficiency.
+
+Ontology forms a cornerstone in the OKP4 protocol, elevating the representation and semantic description of digital resources. Its integration with metadata and commitment to standardization positions OKP4 as a forward-looking protocol in the dynamic landscape of data sharing among professionals.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/resources.md]: Thanks it's corrected in the last commit [4a25fe4](https://github.com/okp4/docs/pull/368/commits/4a25fe4b8b19c7b57ed16042fb5233169e78502a)

**Pull request merged**